src/test/java/com/codurance/allaboard/web/acceptance/UserEngagementSurveyFeature.java

```diff
@@ -73,7 +71,6 @@ void save_a_survey_with_a_description_up_to_1500() {
     assertThat(response.statusCode(), is(201));
     assertThat(response.getContentType(), is(ContentType.JSON.toString()));
     assertThat(responseBody.get("preference"), is(requestBody.get("preference")));
-    assertThat(responseBody.get("email"), is(EMAIL_FROM_TOKEN));
```

Comment:
Why are we taking this assertion out?

Reply:
because the email is something we are not sending anymore from the frontend. All the idea of this PR is to use a view object to don't expose the DB object.,